src/main/java/com/bookbla/americano/domain/notification/service/impl/AlarmServiceImpl.java

```diff
@@ -74,22 +74,53 @@ public PushAlarmCreateResponse sendPushAlarm(PushAlarmCreateRequest pushAlarmCre
 
     @Override
     @Transactional
-    public void sendPushAlarm(Member member, String title, String body) {
+    public void sendPushAlarmForReceivePostCard(Member receiveMember) {
         // 해당 멤버가 푸시 토큰이 없다면 에러 발생
-        if (member.getPushToken() == null) {
+        if (receiveMember.getPushToken() == null) {
             throw new BaseException(PushAlarmExceptionType.NOT_FOUND_TOKEN);
         }
 
         // 해당 멤버가 회원가입 완료상태가 아니면서 매칭 비활성화가 아니라면
-        if (!member.getMemberStatus().equals(MemberStatus.COMPLETED)
-            && !member.getMemberStatus().equals(MemberStatus.MATCHING_DISABLED)) {
+        if (!receiveMember.getMemberStatus().equals(MemberStatus.COMPLETED)
+            && !receiveMember.getMemberStatus().equals(MemberStatus.MATCHING_DISABLED)) {
             throw new BaseException(PushAlarmExceptionType.INVALID_MEMBER_STATUS);
         }
 
-        sendPushAlarmToExpo(member.getPushToken(), title, body);
+        String title = "띵동~\uD83D\uDC8C 엽서가 도착했어요!";
+        String body = receiveMember.getMemberProfile().getName() + "님이 엽서를 보냈어요! 지금 확인해 보세요~\uD83E\uDD70";
```

Comment:
```suggestion
        String body = receiveMember.getMemberProfile().showBlindName() + "님이 엽서를 보냈어요! 지금 확인해 보세요~\uD83E\uDD70";
```

PR 본문 보니까 익명으로 전달되는 것 같은데요, 메서드 만들어놓은게 있습니다!